conda-recipe/build.sh

```diff
@@ -25,6 +25,14 @@ fi
 
 if [ -z "${DALROOT}" ]; then
     export DALROOT=${PREFIX}
+elif [ "${DALROOT}" != "${CONDA_PREFIX}" ]; then
+    # source oneDAL if DALROOT is set outside of conda-build
+    source ${DALROOT}/env/vars.sh
```

Comment:
A few questions:
* Is this part used exclusively by the CI jobs? If not, shouldn't the distributed wheel also ensure that it uses static linkage or that it ships the .so/.dll files and modifies rpath?
* Is `env/vars.sh` always guaranteed to be at that location? I think the pip distributions for example do not have it.

Reply:
https://github.com/intel/scikit-learn-intelex/pull/2125#discussion_r1835789056